src/Tooltip/Tooltip.tsx

```diff
@@ -96,8 +105,25 @@ export default class Tooltip extends React.Component<Props> {
       delayHideMs,
     } = this.props;
 
+    const handleShowTooltip = () => {
+      if (hide) return;
+      setTimeout(() => this.setState({ showTooltip: true }), delayMs);
+    };
+    const handleHideTooltip = () => {
+      if (hide) return;
+      setTimeout(() => this.setState({ showTooltip: false }), delayHideMs);
+    };
+    const handleMouseEnter = clickTrigger ? undefined : handleShowTooltip;
+    const handleMouseLeave = clickTrigger ? undefined : handleHideTooltip;
+    const handleOnClick = this.state.showTooltip ? handleHideTooltip : handleShowTooltip;
```

Comment:
nit: can these callbacks check for `clickTrigger` in the function and no-op if that's the case? Something like 

```
    const handleShowTooltip = () => {
      if (hide || clickTrigger) return;
      setTimeout(() => this.setState({ showTooltip: true }), delayMs);
    };
```

It'll save you the extra assignments and you can directly reference these in the component

Reply:
Don't think we can since onFocus and onBlur use the handleShow/HideTooltip functions and the onMouseEnter/Leave use the handleMouseEnter/Leave functions, since we want onFocus and onBlur to work regardless of whether it's a clickTrigger tooltip or not for keyboard a11y